src/common/checksum.c

```diff
@@ -582,7 +582,9 @@ calc_csum_recx(struct daos_csummer *obj, d_sg_list_t *sgl,
 
 	for (i = 0; i < nr; i++) { /** for each extent/checksum buf */
 		csum_nr = daos_recx_calc_chunks(recxs[i], rec_len, chunk_size);
-		C_TRACE("csum_nr: %lu\n", csum_nr);
+		C_TRACE("Calculating %lu checksum(s) for Array Value [%lu-%lu]",
```

Comment:
rx_idx is uint64_t.   We should use DF_U64 as the print format

e.g: "["DF_U64"-"DF_U64"]"

It's a bit ugly but platform independent.